cmd/root.go

```diff
@@ -45,6 +45,10 @@ func RunCommand() *cobra.Command {
 		Use:   `kit`,
 		Short: shortDesc,
 		Long:  longDesc,
+		// Commands do all their printing directly and return an error only to signal that we should exit with
+		// nonzero status. We don't want to print usage or the error message in this case.
+		SilenceUsage:  true,
```

Comment:
Is this the correct default?

Reply:
Just found out that Cobra is _really_ annoying about this; fixed in 1db15f8 (see my complaining commit message for details)